components/admin_console/system_roles/system_role/system_role_permissions.tsx

```diff
@@ -34,6 +34,11 @@ const sectionsList: SystemSection[] = [
         hasDescription: true,
         subsections: [],
     },
+    {
+        name: 'billing',
+        hasDescription: true,
+        subsections: [],
+    },
```

Comment:
We should only add this section if they are on a cloud installation which can be done by checking the `Cloud` feature in the license

Reply:
Done!